zerver/data_import/rocketchat.py

```diff
@@ -135,11 +135,22 @@ def process_users(
             bot_user["bot_owner"] = realm_owners[0]
 
 
+def truncate_name(name: str, name_id: int, max_length: int = 60) -> str:
+    if len(name) > max_length:
+        idlen = len(str(name_id))
+        name = name[0 : (max_length - 3 - idlen)] + " [" + str(name_id) + "]"
+    return name
```

Comment:
Replaced this with the following for readability:

+    if len(name) > max_length:
+        name_id_suffix = f" [{name_id}]"
+        name = name[0 : max_length - len(name_id_suffix)] + name_id_suffix